evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -419,7 +419,7 @@ public static EVM constantinople(
   }
 
   /**
-   * Constantinople operations registry.
+   * Constantinople operations operation registry.
```

Comment:
nit: "operations operation" reads weird to me

Reply:
Buffalo buffalo buffalo buffalo buffalo.

operation registry for the operations in Constantinople.